decorator.rb

```diff
@@ -0,0 +1,14 @@
+require './Nameable'
+
+class Decorator < Nameable
+  attr_accessor :nameable
+
+  def initalalize(nameable)
+    super()
+    @nameable = nameable
+  end
```

Comment:
- [ ] An error occurs while trying to test out your implementation of decorators as shown below 👇 
![image](https://user-images.githubusercontent.com/46839548/193866037-ec01a252-75a2-491a-bbad-370e5b72e3e0.png)
After prodding around, it realized that it resulted from the typo highlighted. You have `initalalize` instead of `initialize`. Please correct the typo 👍